qa/workunits/rbd/rbd_mirror.sh

```diff
@@ -188,6 +188,14 @@ for i in ${image2} ${image4}; do
   compare_images ${POOL} ${i}
 done
 
+testlog "TEST: snapshot rename"
+snap_name='snap_rename'
+create_snapshot ${CLUSTER2} ${POOL} ${image2} "${snap_name}_0"
+for i in {1..10}; do
+  rename_snapshot ${CLUSTER2} ${POOL} ${image2} "${snap_name}_$(expr ${i} - 1)" "${snap_name}_${i}"
+  wait_for_snap_present ${CLUSTER1} ${POOL} ${image2} "${snap_name}_${i}"
```

Comment:
May be wait only for the last snap present after the loop? I guess it might speed up the test. Or if it makes the test different, may be add both variants?